DiscordBot/Commands/EventCommands.cs

```diff
@@ -340,10 +355,10 @@ private async Task EditEvent(CommandContext context, int eventKey)
 
         [Command("edit")]
         private async Task EditEvent(
-            CommandContext context, 
-            int eventKey, 
-            string editField,
-            DiscordEmbedBuilder? eventEmbed = null)
+            CommandContext context,
+            [Description("Optional key corresponding to event")] int eventKey,
+            [Description("'name', 'description' or 'time'")] string editField,
+            [Description("Ignore this")] DiscordEmbedBuilder? eventEmbed = null)
```

Comment:
Does this actually get printed?

Reply:
Yep unfortunately ?help event edit will pop up all the fields including the embedbuilder field. I just decided for now it would be best to add _some_ kind of description rather than nothing

Reply:
Hm, that's not great... Ideally we'd find a way to make it not print it as an option at all. But as we don't have much time left I'd suggest changing the description to something like "Internal bot use only, please ignore" 🙂

Reply:
Yep that's fine, the reason this is the case is that we're using the default help command. Ideally we'd want to use our own help command which is formatted in a way we want it to be